gulpfile.js/lib/webpack-multi-config.js

```diff
@@ -59,7 +59,7 @@ module.exports = function(env) {
     webpackConfig.output.pathinfo = true
 
     // Create new entries object with webpack-hot-middleware and react-hot-loader (if enabled)
-    if(TASK_CONFIG.javascripts.hot.enabled !== false) {
+    if(TASK_CONFIG.javascripts.hot && TASK_CONFIG.javascripts.hot.enabled !== false) {
```

Comment:
OK, fixed. Moreover addressed a couple more there. Let's see if I didn't mess it up even more :)